// tsparticles-preset-snow/umd/options.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.options = void 0;
    exports.options = {
        background: {
            color: "#333",
        },
        particles: {
            color: {
                value: "#fff",
            },
            move: {
                bounce: false,
                direction: "bottom",
                enable: true,
                random: false,
                straight: false,
            },
            opacity: {
                random: true,
                value: 0.5,
            },
            shape: {
                type: "circle",
            },
            size: {
                random: true,
                value: 10,
            },
        },
    };
});
